### PR TITLE
Record object creation

### PR DESCRIPTION
**Pull request by t1a2l**

### CODE REVIEW

File: app.html [hunk 1]
```diff
@@ -32,21 +33,63 @@
...
+          if (recordArr[0] == "0001")
+            recordObject["ActionType"] = "IN";
+          else
+            recordObject["ActionType"] = "OUT";
+          recordObject["Date"] = DateFormat(recordArr[1]);
+          recordObject["Time"] = TimeFormat(recordArr[2]);
+          recordObject["EmployeeNum"] = parseInt(recordArr[4]);
+          data[recordIndex]["Record"] = recordObject;
+        }
+        document.getElementById('content').innerHTML = JSON.stringify(data);
```
Comment (oferb) [inline app.html]: It's better to return the result of the function, and set this outside.

File: app.html [hunk 2]
```diff
@@ -32,21 +33,63 @@
...
+        console.log(data);
+      }
+      
+      function DateFormat(date) {
```
Comment (oferb) [inline app.html]: Function names should start with lowercase

File: app.html [hunk 3]
```diff
@@ -32,21 +33,63 @@
...
+        year = (parseInt(date[0]) * 10 + parseInt(date[1])) + 2000;
+        month = parseInt(date[2]) * 10 + parseInt(date[3]);
+        if(month < 10)
+          month = "0" + month; 
+        day = parseInt(date[4]) * 10 + parseInt(date[5]);
+        if(day < 10)
+          day = "0" + day; 
+        FormatedDate = day + "/" + month + "/" + year;
```
Comment (oferb) [inline app.html]: No need for an intermediate variable, you can return the result directly.

File: app.html [hunk 4]
```diff
@@ -32,21 +33,63 @@
...
           keyIndex++;
         }       
-
         for (let lineIndex = 1; lineIndex < csv_lines.length; lineIndex++) {
-          entry = csv_lines[lineIndex].split(",");
+          let entry = csv_lines[lineIndex].split(",");
           entries[lineIndex-1] = new Object();
           for (let fieldIndex = 0; fieldIndex < entry.length; fieldIndex++) {
-            entries[lineIndex-1][objectNames[fieldIndex]] = entry[fieldIndex];
+            let UnformatEntry = entry[fieldIndex];
```
[the remark this answers is not in the document]
Comment (t1a2l) [reply app.html]: There is already an entry variable.

File: app.html [hunk 5]
```diff
@@ -4,6 +4,7 @@
 
     <input type="file" id="files" name="files[]" multiple />
     <div id="content"></div>
+    <div id="content1"></div>
```
[the remark this answers is not in the document]
Comment (t1a2l) [reply app.html]: Mistake, used for testing. Removed.

File: app.html (hunk 1 above)
Comment (t1a2l) [reply app.html]: Done

File: app.html (hunk 2 above)
Comment (t1a2l) [reply app.html]: Done

File: app.html (hunk 3 above)
Comment (t1a2l) [reply app.html]: Done

File: app.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (oferb) [reply app.html]: So how about `fieldValue`?

[the remark this answers is not in the document]
Comment (t1a2l) [reply app.html]: OK

**Pull request merged**